toast-tk-plugin-loader/src/main/java/io/toast/tk/plugin/AgentBoot.java

```diff
@@ -0,0 +1,42 @@
+package io.toast.tk.plugin;
+
+import java.lang.instrument.Instrumentation;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.toast.tk.agent.config.AgentConfigProvider;
+
+public class AgentBoot {
+
+	private static final Logger LOG = LoggerFactory.getLogger(AgentBoot.class);
+
+	private static Instrumentation instrumentation;
+
+	/**
+	 * JVM hook to statically load the javaagent at startup.
+	 * 
+	 * After the Java Virtual Machine (JVM) has initialized, the premain method
+	 * will be called. Then the real application main method will be called.
+	 * 
+	 */
+	public static void premain(
+		String args,
+		Instrumentation inst)
+		throws Exception {
+		LOG.info("Premain method invoked with args: {} and inst: {}", args, inst);
+		instrumentation = inst;
+		final String pluginsPath = new AgentConfigProvider().get().getPluginDir();
+		LOG.info("Premain method invoked with pluginDir: {} ", pluginsPath);
+		//PluginLoader.loadPlugins(pluginsPath);
+	}
+
+	public static void agentmain(
+		String agentArgs,
+		Instrumentation inst) {
+		System.out.println("Main method invoked with args: {} and inst: {}");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS106)